OvmfPkg/OvmfPkgX64.dsc

```diff
@@ -50,7 +50,7 @@
   DEFINE NETWORK_IP6_ENABLE             = FALSE
   DEFINE NETWORK_HTTP_BOOT_ENABLE       = FALSE
   DEFINE NETWORK_ALLOW_HTTP_CONNECTIONS = TRUE
-  DEFINE NETWORK_ISCSI_ENABLE           = TRUE
+  DEFINE NETWORK_ISCSI_ENABLE           = FALSE

```

Comment:
What is wrong with simply building your firmware with `-D NETWORK_ISCSI_ENABLE=FALSE` on the command line if you want to omit iSCSI from the build entirely?